cmd/jobs.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/go-playground/validator/v10"
+	"github.com/odpf/guardian/app"
+	"github.com/odpf/guardian/internal/crypto"
+	"github.com/odpf/guardian/jobs"
+	"github.com/odpf/guardian/plugins/notifiers"
+	"github.com/odpf/salt/log"
+	"github.com/spf13/cobra"
+)
+
+func JobsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "jobs",
```

Comment:
@rahmatrhd to make it consistent with rest of the commands, it should be `job` and we can add `jobs` as an alias.